### ISSUE TITLE
Pages (for letter)

### ISSUE THREAD

**Issue by avram** [citation-style-language/zotero-bits#9]
The "# of Pages" (**numPages**) field (mapped to **number-of-pages** in CSL) should be added to _letter_.

Not mentioned in the forums, but this is pretty basic information.

**Comment by rmzelle** [comment]
> The "# of Pages" (numPages) field (mapped to number-of-pages in CSL) should be added to letter.

@avram, are you aware of any style guides that call for this information to be included in references? (I think that should be a minimum requirement for mapping anything to a CSL variable, and it would also help the case for adding Zotero fields)

> It would also make sense to add ... "Medium".

@avram, idem.

> There is an old ticket to add place to letters: https://www.zotero.org/trac/ticket/1565

> Gilbert McMicken to Alexander Morris, 29 November 1881, Glasgow (Scotland), Document 1359, fol. 1r, Alexander Morris Papers, MG-12-84, Provincial Archives of Manitoba, Winnipeg.

@erazlogo, so it looks like we need the "event-place" CSL variable, for "Glasgow (Scotland)". Do we also need the "archive-place" CSL variable for "Winnipeg"? "Provincial Archives of Manitoba" would be the value of the "archive" CSL variable in the example above, right?
(see https://aurimasv.github.io/z2csl/typeMap.xml#map-letter for the current mapping)

**Comment by avram** [comment]
I don't know of style guides asking for number of pages or medium, except perhaps to help with the merge with email that I noted above as well.

That said, I've been far from the world of bibliography for a long time.

**Comment by rmzelle** [comment]
> That said, I've been far from the world of bibliography for a long time.

Yes, I should probably have followed up sooner than 6 years later.

I'm trying to whip these old issues into shape now that Zotero 5.0 is almost ready, and changes to the Zotero data model have been promised to be up next. Without style guide examples this issue might end up on the back-burner, though (with the exception of adding "event-place" for the examples given by @erazlogo above).

**Comment by bwiernik** [comment]
APA requires fields mapped to Medium and Archive.
